jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/corereaders/zos/util/CompressedRecordArray.java

```diff
@@ -494,8 +494,8 @@ private static void fpostest(String filename) throws Exception {
                 }
                 if (eof) break;
             }
-            long now = System.currentTimeMillis();
-            System.out.println("block size " + (1 << blockSizeLog2) + " uses " + ra.memoryUsage() + " took " + (now - then) + " ms");
+            final long now = System.nanoTime();
```

Comment:
This would be more accurate if the difference were computed earlier (before all the `StringBuffer` use):
```
    final long duration = System.nanoTime() - then;
    System.out.println("block size " + (1 << blockSizeLog2) + " uses " + ra.memoryUsage() + " took " + duration + " ns");
```

Please do the same for similar changes.